scipy/signal/filter_design.py

```diff
@@ -1917,20 +1988,39 @@ def iirfilter(N, Wn, rp=None, rs=None, btype='band', analog=False,
 
     Examples
     --------
-    Generate a 17th-order Chebyshev II bandpass filter and plot the frequency
-    response:
+    Generate a 17th-order Chebyshev II analog bandpass filter from 50 Hz to
+    200 Hz and plot the frequency response:
 
     >>> from scipy import signal
     >>> import matplotlib.pyplot as plt
 
-    >>> b, a = signal.iirfilter(17, [50, 200], rs=60, btype='band',
-    ...                         analog=True, ftype='cheby2')
+    >>> b, a = signal.iirfilter(17, [2*np.pi*50, 2*np.pi*200], rs=60,
```

Comment:
Scaling these frequencies by 2*pi is not necessary.  This example of an analog filter was fine the way it was.

Reply:
OK, my comment stands.  There is no fixed absolute scale for an analog filter, so no need to convert whatever the input units are to rad/s.

If you prefer to interpret the values in the example as Hz, the only change needed in the example is to change the label of the `x` axis to `Frequency [Hz]`.

Reply:
Yeah I changed it so it's equivalent to the following digital example

Reply:
OK, to be consistent about using Hz, for the analog example, you can leave `Wn` as `[50, 200]` and just change the label of the `x` axis as I mentioned above.  There is no need for converting back and forth to rad/s; that will just confuse readers.

Reply:
> There is no fixed absolute scale for an analog filter

@WarrenWeckesser But the b, a coefficients represent a transfer function that's based on angular frequency, not ordinary frequency, right?  It's not just the input frequencies and output plot that matter for the example, it's the intermediate values too

Reply:
With an analog design, if you change the units of the input frequencies, the coefficients change, but only to account for the change in scale.  The coefficients are not based on any specific units.  For example,

```
In [80]: b, a = signal.iirfilter(5, [50, 200], rs=60, btype='band', analog=True, ftype='cheby2')

In [81]: a
Out[81]: 
array([1.00000000e+00, 2.08556501e+02, 7.17476259e+04, 9.75002759e+06,
       1.70949761e+09, 1.54504252e+11, 1.70949761e+13, 9.75002759e+14,
       7.17476259e+16, 2.08556501e+18, 1.00000000e+20])
```
Multiply both input frequencies by 2*pi and design the filter again:

```
In [82]: b2, a2 = signal.iirfilter(5, [2*np.pi*50, 2*np.pi*200], rs=60, btype='band', analog=True, ftype='cheby2')

In [83]: a2
Out[83]: 
array([1.00000000e+00, 1.31039914e+03, 2.83248274e+06, 2.41849643e+09,
       2.66432973e+12, 1.51300296e+15, 1.05183522e+18, 3.76933662e+20,
       1.74279571e+23, 3.18304331e+25, 9.58956006e+27])
```
`a2` looks different, but that's because the coefficients include powers of the scale factor 2*pi.  That scale can be taken out to recover `a`:
``` 
In [84]: a2/(2*np.pi)**np.arange(0, len(a))
Out[84]: 
array([1.00000000e+00, 2.08556501e+02, 7.17476259e+04, 9.75002759e+06,
       1.70949761e+09, 1.54504252e+11, 1.70949761e+13, 9.75002759e+14,
       7.17476259e+16, 2.08556501e+18, 1.00000000e+20])
```
So unlike the digital case, which has a sample period that determines a scale, there is no natural scale for the analog case.  As long as you are consistent, everything will just work.  If you specify the design frequencies using, say, gradians/fortnight, and then evaluate the frequency response at a set of frequencies expressed in gradians/fortnight, it will work.

Reply:
But the transfer function coefficients are based on the Laplace transform and the definition *s = σ + jω*, which is angular frequency.  

For example, `iirfilter(1, 1, btype='low', analog=True)` produces `(array([1.]), array([1., 1.]))` which represents the transfer function H(s) = 1/(s+1), which has a −3 dB point at 1 rad/unit.

I've never seen anyone define *s* using ordinary frequency, and I don't think we should do that just to make the example look simpler.  The docstring examples should demonstrate how `iirfilter` is normally used, which is angular frequency, and introducing scale factors will just create confusion.

Reply:
@endolith, OK, my extended brain fart is over.  You're right, the continuous time functions should have frequencies expressed in rad/[time unit].  Thanks for your patience!

Reply:
> I've never seen anyone define s using ordinary frequency, 

Not that I want to argue against but we actually started exactly doing that because rad/s is just a scalar too much and means nothing in continuous time. First thing we all do, in matlab, for example is to switch to Hz. because that's what we are measuring and seeing on the plot. Same argument holds for dB. The extra 20 factor means nothing to almost(!) anyone.

Anyways, I guess we should stick to the conventions so I would also choose rad/s here. 

The main reason for my comment is (😃) that we started to see the GitHub unicorn. Do you mind maybe squashing some of the earlier commits such that the page is a bit less dense?

Reply:
I don't even know if that would help but just to keep the page still loadable worth a try I guess.